### PR TITLE
Add manage vault members screen

### LINKED ISSUE #6790: [Groups UI] Support add members / table in vaults

### PR DESCRIPTION
**Pull request by tdraier**
fixes: https://github.com/dust-tt/dust/issues/6790

## Description

- Add members table and handle the "Add members" modal. Add members modal allows to add or remove members from the group (same display as when creating a vault).
- Members can also be removed by going into the more menu

Capture-2024-08-23-104637
Capture-2024-08-23-104723
Capture-2024-08-23-104758

## Risk

none

## Deploy Plan

deploy `front`

### CODE REVIEW

File: front/components/vaults/CreateVaultModal.tsx [hunk 1]
```diff
@@ -77,34 +75,38 @@ export function CreateVaultModal({
           const isSelected = selectedMembers.includes(info.row.original.userId);
           if (isSelected) {
             return (
+              <div className="full-width flex justify-end">
```
Comment (tdraier) [inline front/components/vaults/CreateVaultModal.tsx:78]: just added a div here for better padding

File: front/components/vaults/VaultMembers.tsx [hunk 2]
```diff
@@ -13,13 +43,57 @@ type VaultMembersProps = {
...
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        memberIds,
+      }),
+    });
+
+    if (!res.ok) {
+      const errorData = await res.json();
+      console.log("errorData", errorData);
```
Comment (PopDaph) [inline front/components/vaults/VaultMembers.tsx:69]: Send error notif?

File: front/components/vaults/VaultMembers.tsx [hunk 3]
```diff
@@ -1,10 +1,40 @@
...
+
+const tableColumns = [
+  {
+    id: "name",
+    cell: (info: CellContext<RowData, string>) => (
+      <DataTable.CellContent avatarUrl={info.row.original.icon}>
+        {info.getValue()}
+      </DataTable.CellContent>
+    ),
+    accessorFn: (row: RowData) => row.name,
```
Comment (PopDaph) [inline front/components/vaults/VaultMembers.tsx:34]: Does this will sort by name by default?

**Review by PopDaph — APPROVED** [review]
👍🏻

**Pull request merged**